src/components/Login.tsx

```diff
@@ -55,6 +116,24 @@ const Login = (props: Props) => {
     <div>
       <Stack align="center" spacing="xl">
         <div style={{ width: 280 }}>
+          {!isInitialized && (
+            <>
+              <TextInput 
+                label="First Name" 
+                placeholder='Jhon' 
+                radius="md" 
+                size="md" 
+                onChange={onFirstNameChange}
+              />
+              <TextInput 
+                label="Last Name" 
+                placeholder='Doe' 
```

Comment:
Maybe placeholder for first name and last name to be `Satoshi` and `Nakamoto` 🦀  ?